core/src/html/text_format.rs

```diff
@@ -717,7 +717,34 @@ impl FormatSpans {
                             }
 
                             if let Some(size) = attribute(b"size") {
-                                format.size = size.parse().ok();
+                                // if the numer starts with + or -, the size is relative
+                                let (prefix, size) = if size.starts_with(&[b'+', b'-'][..]) {
```

Comment:
To be sure, how does this work? And what does `[..]` do here?
Intuitively, with slice/str APIs, pretty sure this would be equivalent to `starts_with(b"+-")`, which checks for literal prefix "+-", not "+ or -".

Reply:
According to the `Pattern` docs:

> - `&[u8]` searches for any of the given LATIN1 code units.

And `[..]` is to make a slice of `u8` (to fulfill the `&[u8]` requirement). I did it this way because I saw it somewhere, I'm not sure if there's a clearer way

Reply:
Oh OK, I think I get it now.
Anyway, I think `&[b'+', b'-']` is already a `&[u8]`? Does this work without the `[..]`?

Reply:
That's exactly my line of thought I had when I first saw this syntax :) Unfortunately `&[b'+', b'-']` is `&[u8, 2]` and it does not work.

I basically have to give a hint to rustc that it should be `&[u8]`. Alternatively, I could do `&[b'+', b'-'] as &[u8]`, but the `[..]` syntax was used multiple times throughout ruffle, and the `as` version has never been used.